packages/sdk/package.json

```diff
@@ -9,16 +9,14 @@
         "build": "tsc",
         "cb": "yarn clean && yarn build",
         "clean": "rm -rf dist",
-        "dev": "my-node src/index.ts",
         "lint": "yarn eslint --format unix ./src  --max-warnings=0",
         "lint:fix": "yarn lint --fix",
-        "my-jest": "NODE_EXTRA_CA_CERTS=${HOME}/river-ca-cert.pem yarn node -r ../../scripts/node-no-warn.js --experimental-vm-modules $(yarn bin jest) --testTimeout=300000",
-        "my-node": "NODE_EXTRA_CA_CERTS=${HOME}/river-ca-cert.pem yarn node -r ../../scripts/node-no-warn.js --loader ts-node/esm --experimental-specifier-resolution=node --experimental-vm-modules",
-        "test": "yarn test:ci:multi:ne",
-        "test:ci": "yarn test:ci:multi:ne",
-        "test:ci:multi:ent:legacy": "USE_LEGACY_SPACES=true RIVER_ENV=local_multi yarn my-jest --group=with-entitlements",
-        "test:ci:multi:ent": "RIVER_ENV=local_multi yarn my-jest --group=with-entitlements --group=with-v2-entitlements",
-        "test:ci:multi:ne": "RIVER_ENV=local_multi_ne yarn my-jest --group=main",
+        "test": "yarn test:multi:ne",
+        "test:watch": "vitest --watch",
+        "test:ci": "yarn test:multi:ne",
+        "test:multi:ent-legacy": "vitest run --config vitest.config.entitlements-legacy.ts",
+        "test:multi:ent": "vitest run --config vitest.config.entitlements.ts",
+        "test:multi:ne": "vitest run --config vitest.config.ts",
```

Comment:
1) why did you rename these jobs? can we leave the names the same at least?
2) we have --groups that we were selecting the tests that had a ` * @group main` or ``* @group with-entitlements` I guess the filename convention is okay for this? but i don't see that you renamed the tests in the syncAgent folder so maybe you missed those? And if we're going to have custom names I would rather make a custom extension for NO Entitlements than for entitlements, entitlements should be the default moving forward.
`include: ['./src/**/*.test.entitlements.ts', './src/**/*.test.entitlements-v2.ts'],`

Reply:
myTest.test.unentitled.ts

Reply:
Vitest doesn't have `--group` support, that's why we need to rename the test itself or make a separate folder like `/sdk/src/tests/no-entitlements/*.test.ts`

I'll revert the test job name back to the original.
Also thanks for the catch about sync agent tests, I'll add it later.

Reply:
is that right?
- `group main` -> `myTest.test.noentitlements.ts`
- `group entitlement` -> `myTest.test.ts`
- `group v2-entitlement` -> `myTest.test.entitlement-v2.ts`

Reply:
the names are wrong, but it's directionally correct. However, it's ugly. What if we do this
-`no group` -> `myTest.test.ts` runs with `test:unit` (we need to add a new scripts entry `test:unit` for this in sdk)
- `@group main` -> `myTest.test.multi_ne.ts`
- `@group with-entitlements` -> `myTest.test.multi.ts`
- `@group with-v2-entitlements` => `myTest.test.multi.v2.ts`

now they match the environment names
I think @clemire had a plan for the v2 entitlements test? maybe she has an opinion.

Reply:
done, lmk if it matches the expectation

Reply:
I like folder for test group idea: clear what's going on, easy to follow.